amlip_docs/rst/user_manual/nodes/inference.rst

```diff
@@ -0,0 +1,65 @@
+.. include:: ../../exports/alias.include
+
+.. _user_manual_nodes_inference:
+
+##############
+Inference Node
+##############
+
+This node waits for data serialized as :ref:`user_manual_datatype_inference`, and once received it calculate the inference whose output is the inference solution as :ref:`user_manual_datatype_inference_solution`.
+
+.. warning::
+
+    The use of an Inference node must be called once for each inference to be processed.
+    This means that once the data is sent, the thread must wait for the inference to arrive before sending another data.
+    Asynchronous methods are available on the :ref:`user_manual_nodes_async_inference`.
+
+
+Example of Usage
+================
+
+This node kind does require **active** interaction with the user to perform its action.
+User can use method :code:`request_inference` to send new data.
+The thread calling this method will wait until the whole process has finished and the *Inference* has arrived from
+the *Inference Node* in charge of this data.
+By destroying the node every internal entity is correctly destroyed.
+
+Steps
+-----
+
+* Instantiate the Inference Node creating an object of such class with a name.
+* Wait for the data by calling :code:`process_inference`.
+* Return the inference as an :code:`InferenceSolutionDataType`.
+
+.. tabs::
+
+    .. tab:: C++
+
+        .. code-block:: cpp
+
+            // Create a new Inference Node
+            auto node = eprosima::amlip::InferenceNode("My_Inference_Node");
+
+            // Create a callback to process data and return the inference
+            auto engine_routine = []( const eprosima::amlip::types::InferenceDataType& ){
```

Comment:
add the name of the variable, `dataset` for example, or `to_inference_value`

Same in Async